tests/cupyx_tests/scipy_tests/sparse_tests/test_coo.py

```diff
@@ -155,6 +156,14 @@ def test_init_copy_other_scipy_sparse(self, xp, sp):
         assert n.shape == m.shape
         return n
 
+    def test_pickle_roundtrip(self, xp, sp):
+        s = _make(xp, sp, self.dtype)
```

Comment:
I think you just need this? 
```suggestion
    def test_pickle_roundtrip(self):
        s = _make(cupy, sparse, self.dtype)
```
(Likewise for all roundtrip tests.)

Reply:
Huh interesting. I thought this would be tested with NumPy/SciPy first and then CuPy second. Is there a way for me to get that behavior here?

Reply:
You can do a quick loop like this: https://github.com/cupy/cupy/blob/7c96b16fe26277f383ebde190cc36d5518940565/tests/cupyx_tests/scipy_tests/sparse_tests/test_coo.py#L491-L492
But what's the point? Don't you just wanna test if CuPy sparse matrices are picklable? You can't compare CuPy's pickle with SciPy's, after all.

Reply:
Good point.

Reply:
Fixed in the latest commit 🙂